tests/testthat/test-shinytest2-data_summary.R

```diff
@@ -107,6 +107,8 @@ testthat::test_that(
       }
     )
 
+    datanames(data) <- c("mtcars1", "mtcars2", "iris", "miniACC", "CO2")
```

Comment:
```suggestion

```

Reply:
You were trying to remove this warning?

```r
── Warning: e2e: data summary is displayed properly if teal_data include data.frames with join keys, MAE objects and vectors ──
`data` object has no datanames. Default datanames are set using `teal_data`'s environment.
Backtrace:
    ▆
 1. └─TealAppDriver$new(data = data, modules = example_module())
 2.   └─teal (local) initialize(...)
 3.     └─teal::init(...) at teal/R/TealAppDriver.R:55:7
```

What about `suppressWarnings()` ? Or settings datanames but with `factors` included in the vector of names.

Reply:
Yes, I wanted to remove the warnings. Sure we could use `suppressWarnings`. I just went with assigning datanames because we do the same thing in other tests.
I see that we're checking for the presence of `factors` so I'll include this as well.

```suggestion
    datanames(data) <- c("mtcars1", "mtcars2", "iris", "miniACC", "CO2", "factors")
```